crates/matrix-sdk-crypto/src/olm/group_sessions/inbound.rs

```diff
@@ -60,25 +60,85 @@ use crate::{
 // sessions that were created between some time period, this should only be set
 // for non-imported sessions.
 
-/// Inbound group session.
+/// Information about the creator of an inbound group session.
+#[derive(Clone)]
+pub(crate) struct SessionCreatorInfo {
+    /// The Curve25519 key of the session creator.
+    ///
+    /// If the session was received directly from the creator through an
+    /// `m.room_key` event, this key corresponds to the long-term Curve25519
+    /// identity key of the Olm Session that was utilized to decrypt the
+    /// event.
```

Comment:
```suggestion
    /// If the session was received directly from its creator device through an
    /// `m.room_key` event (and therefore, session sender == session creator),
    /// this key equals the Curve25519 device identity key of that device. Since
    /// this key is one of three keys used to establish the Olm session through
    /// which encrypted to-device messages (including `m.room_key`) are sent,
    /// this constitutes a proof that this inbound group session is owned by
    /// that particular Curve25519 key.
```